<?xml version="1.0" encoding="UTF-8"?>  <!--xml 버전과 글자 버전 종류-->
<ui version="4.0">                      <!-- ui 버전-->
 <class>MainWindow</class>              <!-- 클래스의 이름 -->
 <widget class="QMainWindow" name="MainWindow">  <!-- <widget> : UI의 한 요소(위젯)를 나타냄 / class="QMainWindow" : 이 위젯의 타입이 QMainWindow 이라는 것을 말함 / 이 객체의 이름(식별자)입니다.-->
  <property name="geometry">  <!--Qt Designer에서 만든 UI 위젯(버튼, 창, 라벨 등)의 위치와 크기를 정의하는 속성이며 그 이름을 정의 하는 것 -->
   <rect>
    <x>0</x>  <!-- 위치 x 값-->
    <y>0</y>  <!-- 위치 y 값-->
    <width>500</width>  <!-- 넓이 값-->
    <height>300</height> <!-- 높이 값-->
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weather App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="city_input">
      <property name="placeholderText">
       <string>도시 이름을 입력하세요</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_weather">
      <property name="text">
       <string>날씨 조회</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="result_label">
      <property name="text">
       <string>날씨 결과가 여기에 표시됩니다</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
